### PR TITLE
Buying item test

### PR DESCRIPTION
**Pull request by KonradSusel**
This test tests if logic responsible for deciding if user has enough money to buy item is working correctly.

### CODE REVIEW

File: src/tests/java/CodecoolerDAOTest.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.mockito.Mockito.*;
+
+
+class CodecoolerDAOTest {
+
+    @Test
+    void testIfCheckingIfCodecoolerHasEnoughMoneyToBuyItemWorksCorrectly() throws IOException {
```
Comment (Kamillus89) [inline src/tests/java/CodecoolerDAOTest.java]: testIfCheckingIf.... - it has no sens, isn't better without :
testIfCodecooler... 
?

Comment (ejaworska) [reply src/tests/java/CodecoolerDAOTest.java]: Yes, that's right !

**Review by Kamillus89 — CHANGES_REQUESTED** [review]
You could also do some refactor for this test - for example extract some methods from test method for better clean code.

File: src/tests/java/CodecoolerDAOTest.java (hunk 1 above)
Comment (KonradSusel) [reply src/tests/java/CodecoolerDAOTest.java]: But I am not checking if codecooler has enough money to buy item. I am checking if logic responsible for checking that works correctly :( I don't understand.

Comment (Kamillus89) [reply src/tests/java/CodecoolerDAOTest.java]: so better it would be:
testIfCodecoolerHasEnoughMoneyMethodWorksCorrectly

**Review by ejaworska — COMMENTED** [review]
It looks good.

**Pull request closed without merging**